### ISSUE TITLE
Latent Effects don't load when equipped and zoning

### ISSUE THREAD

**Issue by Skerxan** [DerpyProjectGroup/topaz#1081]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened
- [x] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)
- [x] I have specified what branch this happens on branch: EDITME

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

We had several reports of items like CoP rings, Destrier Beret and Shadow Mantle without active latent effects even when they should be active.
It happens when you zone somewhere or log-in.
This gets usually fixed by re-equipping the gear.

**Comment by TeoTwawki** [comment]
I haven't seen this yet, but it may be that the specific latents used on your server aren't in use don mine..except CoP rings. My CoP rings are def workin on zone !

**Comment by Skerxan** [comment]
I also had this tested and reported on another server ages ago which closed down.

Try having any stats-increasing latents equipped and relog. See if they are active when you log-in. Then re-equip them. See the changes.

**Comment by personaone** [comment]
yup actually can confirm with ninja boots during nighttime, zoning  with them on actually doesn't kick in the latent until you re-equip the item, it's at night of course.

**Comment by TeoTwawki** [comment]
I am wondering if we flubbing the packets at zonetime so the client isn't being informed, or if server side the latent code is truly not being called when the entity is rebuilt on the other side..

These -were- working not all that long ago from my perspective. Not sure whats changed.

**Comment by Skerxan** [comment]
There seems to be another bug that may relate to this one.
It seems that latents stay active even if their triggering condition is gone again. Until you zone atleast.
This has been the case with Destrier Berets lvl 30 condition. It would still give the player the stats above lvl 30.
Or i am entirely wrong and its just a problem with their mods or something else. Didn't dig deeper into it yet.
